js/long-form.js

```diff
@@ -0,0 +1,70 @@
+// /**
+//  * Created by pchotrani on 17/03/16.
+//  */
+
+(function ($) {
```

Comment:
Comment added,
Have a read https://learn.jquery.com/plugins/basic-plugin-creation/#protecting-the-alias-and-adding-scope.